mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(boolean) {
+  let isHappy = boolean;
```

Comment:
you were on the right way
to make it even more better you should make `isHappy` the function's parameter:
```js
function getMood(boolean) {
   if (isHappy) {
      ....
   } else {
      ....
   }
}
```

for the future, never give names to your variables same as keywords or datatypes.
`boolean` is the datatype in JS: [Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
in this particular case it might work for you, coz you named it starting with small `b`, 
but it's still a Data Type. Don't use `boolean`  word variable names.